tests/pass-dep/tempfile.rs

```diff
@@ -19,3 +20,24 @@ fn test_tempfile_in() {
     let dir_path = utils::tmp();
     tempfile::tempfile_in(dir_path).unwrap();
 }
+
+fn test_tempfile_ext() -> std::io::Result<()> {
+    use std::io::Write;
+    use std::os::unix::fs::FileExt;
+
+    let mut f = tempfile::tempfile()?;
+    f.write_all(b"hello world")?;
+    let mut buf = [0u8; 3];
+    f.read_exact_at(&mut buf, 2)?;
+    assert_eq!(&buf, b"llo");
+    f.read_exact_at(&mut buf, 6)?;
+    assert_eq!(&buf, b"wor");
```

Comment:
Can you add some normal `read`s to ensure that read_exact_at leaves the file offset unaffected?